include/BoolType.hh

```diff
@@ -2,10 +2,9 @@
  * @file BoolType.hh
  * @author Caleb Aikens (caleb@distributive.network)
  * @brief Struct for representing python bools
- * @version 0.1
  * @date 2022-12-02
  *
- * @copyright Copyright (c) 2022
+ * @copyright Copyright (c) 2024 Distributive Corp.
```

Comment:
we shouldn't update those unless they were touched in 2024 after 2022. Please revert all similar ones

Reply:
reverted